backend/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+services:
+
+  mysqldb:
+    image: mysql:8
+    container_name: mysqldb
+    ports:
+      - 3305:3306
+    restart: unless-stopped
+    volumes:
+      - db:/var/lib/mysql
+    environment:
+      MYSQL_DATABASE: 'SkyNest'
+      MYSQL_USER: 'user'
+      MYSQL_PASSWORD: '123456'
+      MYSQL_ROOT_PASSWORD: '123456'
+
+  cassandra:
```

Comment:
Why would you use Cassandra? What is the reason behind it?

Reply:
We'll need to store invalid tokens somewhere until they expire. A NoSQL database is probably a good choice.

Cassandra has an easy solution for this: any inserted row can have a time-to-live defined, so tokens can live in the database just until they expire.
I've also managed to make this work in code using the Cassandra implementation of CrudRepository, but there's a bit of a workaround involved.

I think MongoDB has something similar - Cassandra just seemed more straightforward in this regard.

Reply:
Interesting, I would like to see this in action ;)